app/src/main/java/ch/epfl/sdp/dependencies/AppContainer.java

```diff
@@ -32,4 +35,8 @@
     public ServerDatabaseAPI serverDatabaseAPI = new ServerFirestoreDatabaseAPI();
 
     public ClientDatabaseAPI clientDatabaseAPI = new ClientFirestoreDatabaseAPI();
+
+    public Market marketBackend = null;

```

Comment:
Having the market backend stored in the AppContainer makes it impossible to create two markets at the same time right? Would it be possible to take it from the Server since it creates it?